app/Http/Controllers/API/TraderController.php

```diff
@@ -152,9 +151,11 @@ public function showVoucherHistory(Trader $trader)
      */
     public function emailVoucherHistory(Request $request, Trader $trader)
     {
+        $vouchers = $trader->vouchersConfirmed;
+        $title = 'A report containing voucher history.';
         // Request date string as dd-mm-yyyy
         $date = $request->submission_date ? $request->submission_date : null;
```

Comment:
There's a neat thing in PHP 7 called the null coalesce operator which makes this kind of assignment a lot easier.
This:
`$date = $request->submission_date ?? null;` would be the equivalent I believe. If it's against the code style for this project ignore me.